tools/gulptasks/dashboards/dlint.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) Highsoft AS
+ */
+
+const gulp = require('gulp');
```

Comment:
We already have this tested in the `tools/gulptasks/dashboards/test.js` so there is no need to duplicate that.

Besides that, this task is not being used at all.

Reply:
Indeed! I was not aware of it, let's optimize it.